server/migrations/20240305205951-removeAtMode.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+    async up(queryInterface, Sequelize) {
+        return queryInterface.sequelize.transaction(async transaction => {
+            await queryInterface.dropTable('AtMode', {
+                cascade: true,
+                transaction
+            });
+
+            const testPlanVersions = await queryInterface.sequelize.query(
+                `SELECT id, tests FROM "TestPlanVersion"`,
+                {
+                    type: Sequelize.QueryTypes.SELECT,
+                    transaction
+                }
+            );
+            await Promise.all(
+                testPlanVersions.map(async ({ id, tests }) => {
+                    const updatedTests = JSON.stringify(
+                        tests.map(test => {
+                            delete test.atMode;
+                            return test;
+                        })
+                    );
+                    await queryInterface.sequelize.query(
+                        `UPDATE "TestPlanVersion" SET tests = ? WHERE id = ?`,
+                        { replacements: [updatedTests, id], transaction }
+                    );
+                })
+            );
```

Comment:
```suggestion
            );

            await regenerateResultsAndRecalculateHashes(
                queryInterface,
                transaction,
                { pruneOldVersions: false }
            );
```

You'll also need a migration to update the `hashedTests` field of TestPlanVersion since the atMode was factored in the tests hash. Otherwise when doing a new import, there will be duplicates of what's already in the system